app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,47 @@
+<section class='container'>
+  <h1><%= @recipe.name %></h1>
+  <div class='subcontainer'>
+    <div class='column-left'>
+      <p class='biginfo'>Preparation Time: <%= @recipe.preparation_time %> hs.</p>
+      <p class='biginfo'>Cooking Time: <%= @recipe.cooking_time %> hs.</p>
+      <p> <%= @recipe.description %> </p>
+      <%= link_to 'Generate Shopping List', shopping_list_path(@recipe.id), class: :linkbutton %>
+    </div>
+    <% if can? :manage, @recipe%>
+    <div class='column-right'>
+      <div class='togglediv'>
+        <p class='togglelabel'>Public</p>
+        <%= link_to recipe_toggle_path(@recipe.id), data: { "turbo-method": :post } do %>
+          <label class="switch" checked='<%= @recipe.public %>'>
+            <input type="checkbox" checked='<%= @recipe.public %>'>
+            <span class="slider round" checked='<%= @recipe.public %>'></span>
+          </label>
+        <% end %>
+      </div>
+      <%= link_to 'Add Ingredient', ingredient_new_path(@recipe.id), class: :linkbutton %>
+      <% end %>
+    </div>
+  </div>
+  <span></span>
+  <table>
+    <tr>
+      <th>Food</th>
+      <th>Quantity</th>
+      <th>Value</th>
+      <th>Actions</th>
+    </tr>
+    <% @recipefoods.each do |recipefood| %>
+      <tr>
+        <td><%= recipefood.food.name %></td>
+        <td><%= recipefood.quantity %> <%= recipefood.food.measurement_unit %></td>
+        <td>$ <%= recipefood.quantity * recipefood.food.price %></td>
+        <td>
+          <% if can? :manage, @recipe%>
+            <%= button_to 'Delete', ingredient_delete_path(recipe_id: @recipe.id, id: recipefood.id), method: :delete %>
+            <%= link_to 'Modify', ingredient_edit_path(recipe_id: @recipe.id, id: recipefood.id) %>
+          <% end %>
+        </td>
+      </tr>
```

Comment:
- [ ] @asdt560 and @d4nQw3rty good work on your views however it's best practice to use partial views for your recipefoods, kindly add it as you did for the foods.